scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -271,6 +283,15 @@ export function RelatedItemsDirective(authoringWorkspace: AuthoringWorkspaceServ
                     scope.refreshRelatedItems();
                 }
             });
+
+            const removeEventListeners = [
+                'item:lock',
+                'item:unlock',
+            ].map((_) => $rootScope.$on(_, scope.refreshRelatedItems));
+
+            scope.$on('$destroy', () => {
+                removeEventListeners.forEach((_) => _());
```

Comment:
What's up with the underscores? :D The convention is that underscores are used for argument names only when you don't use the argument.

Reply:
But it looks cool doesn't it? Haha I'll change it 👍